packages/product-service/src/handlers/getProductsById.js

```diff
@@ -0,0 +1,32 @@
+import * as db from "../db";
+
+export async function handler(event) {
+  const headers = {
+    "Access-Control-Allow-Origin": "*",
+    "Access-Control-Allow-Credentials": true,
+  };
+
+  if (!event?.pathParameters?.productId) {
```

Comment:
Sorry, I believe there is no need to add question mark for event and pathParameters. As per API they should always be truthy. Please correct me if I'm wrong